include/odp/api/spec/ipsec.h

```diff
@@ -1206,6 +1210,12 @@ typedef struct odp_ipsec_status_t {
  * restored. The amount and content of packet data before the IP header is
  * undefined.
  *
+ * Additional TFC padding might be present after packet contents for ESP tunnel
```

Comment:
I interpret the "might be present" so that if there was TFC padding in a received packet, it might or might not be present after ODP IPsec processing. Is that the intended interpretation and does it need to be made clearer?

Reply:
Well. RX side has no way to know in advance if there will TFC padding or not in the packet. That is why I used "might" here. Could you please suggest better wording?

Reply:
So you are saying that my interpretation was not the intended one. I think it is important to make it clear whether we mean that there might of might not be padding in the received packet in the first place or whether the implementation might or might not strip said padding when present.

I was thinking it would be good to not make any promises on whether the padding is removed or not, i.e. specify that there may or may not be padding left in the result packet that the application gets. This would work with any implementation, even those that do not always do the same thing for every packet (e.g. depending on the input port).

Not promising anything about the padding would be similar to how the API currently leaves it up to the implementation whether there is any L2 or other bytes in front of the IP header in the ODP packet data.

Anyway, I will not attempt better wording before it is clear what the intended semantics are.

Reply:
I have to think about it.